x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -102,6 +102,7 @@ export function getPluginPrivileges() {
       ...privilege,
       api: userMlCapabilitiesKeys.map((k) => `ml:${k}`),
       catalogue: [PLUGIN_ID],
+      management: { insightsAndAlerting: [] },
```

Comment:
@elastic/ml-ui I took a guess here based on how your functional tests are written, but I could use confirmation:

Do you expect the `user`/`read` privileges to not grant access to the ML **Management** app?
To say another way, do you have to be an ml `admin` (the `all` privilege) in order to access this?